app/controllers/lambda_callback/address_proof_result_controller.rb

```diff
@@ -4,6 +4,11 @@ def create
       dcs = DocumentCaptureSession.find_by(result_id: result_id_parameter)
 
       if dcs
+        analytics.track_event(
+          Analytics::LAMBDA_RESULT_ADDRESS_PROOF_RESULT,
+          result: address_result_parameter,
```

Comment:
are we trying to log the whole result? or just the result ID?

Reply:
Do we want to guard this on env so we don't create too many extra log events?

Reply:
I think these could good to have even beyond smoke test issues. I don't think they'd be too numerous since they'd be limited to proofing

We do log full responses, at least for errors, during phone verification from what I can see in cloudwatch. There isn't any PII in the lambda callbacks with the exception of doc auth, which should be explicitly excluding it.

Reply:
makes sense!